utils/convert_srg_export_to_xlsx.py

```diff
@@ -0,0 +1,118 @@
+    sheet.column_dimensions['O'].width = 3 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['P'].width = 3 * sheet.column_dimensions['A'].width
+    sheet.column_dimensions['Q'].width = 3 * sheet.column_dimensions['A'].width
+
+
+def setup_headers(sheet: openpyxl.worksheet.worksheet.Worksheet) -> None:
+    sheet['A1'] = 'IA Control'
+    sheet['B1'] = 'CCI'
+    sheet['C1'] = 'SRGID'
+    sheet['D1'] = 'STIGID'
+    sheet['E1'] = 'SRG Requirement'
+    sheet['F1'] = 'Requirement'
+    sheet['G1'] = 'SRG VulDiscussion'
+    sheet['H1'] = 'Vul Discussion'
+    sheet['I1'] = 'Status'
+    sheet['J1'] = 'SRG Check'
+    sheet['K1'] = 'Check'
+    sheet['L1'] = 'SRG Fix'
+    sheet['M1'] = 'Fix'
+    sheet['N1'] = 'Severity'
+    sheet['O1'] = 'Mitigation'
+    sheet['P1'] = 'Artifact Description'
+    sheet['Q1'] = 'Status Justification'
+    for cell in list(sheet.iter_rows(max_row=1))[0]:
+        cell.font = Font(bold=True, name='Calibri')
+
+
+def format_cells(sheet: openpyxl.worksheet.worksheet.Worksheet):
+    for row in sheet.iter_rows():
+        for cell in row:
+            cell.alignment = cell.alignment = Alignment(wrap_text=True, vertical='top')
+
+
+def setup_row(sheet: openpyxl.worksheet.worksheet.Worksheet, row: dict, row_num: int) -> None:
+    sheet[f'A{row_num}'] = row['IA Control']
+    sheet[f'B{row_num}'] = row['CCI']
+    sheet[f'C{row_num}'] = row['SRGID']
+    sheet[f'D{row_num}'] = row['STIGID']
+    sheet[f'E{row_num}'] = row['SRG Requirement']
+    sheet[f'F{row_num}'] = row['Requirement']
+    sheet[f'G{row_num}'] = row['SRG VulDiscussion']
+    sheet[f'H{row_num}'] = row['Vul Discussion']
+    sheet[f'I{row_num}'] = row['Status']
+    sheet[f'J{row_num}'] = row['SRG Check']
+    sheet[f'K{row_num}'] = row['Check']
+    sheet[f'L{row_num}'] = row['SRG Fix']
+    sheet[f'M{row_num}'] = row['Fix']
+    sheet[f'N{row_num}'] = row['Severity']
+    sheet[f'O{row_num}'] = row['Mitigation']
+    sheet[f'P{row_num}'] = row['Artifact Description']
+    sheet[f'Q{row_num}'] = row['Status Justification']
+    sheet.row_dimensions[row_num].height = 42
+
+    # freeze header row represented by A1
+    # A2 is required because it freezes everything before it
+    # in this case we only want A1 row to be frozen
+    sheet.freeze_panes = "A2"
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-o', '--output', type=str,
+                        help=f"The path to the output. Defaults to {OUTPUT}",
+                        default=OUTPUT)
+    parser.add_argument("-r", "--root", type=str, action="store", default=SSG_ROOT,
+                        help=f"Path to SSG root directory (defaults to {SSG_ROOT})")
+    parser.add_argument('input', type=str)
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    xlsx = openpyxl.Workbook()
+    sheet = xlsx.active
+    sheet.name = 'Sheet0'
```

Comment:
Let's add something meaningful here. For example, `<Product> SRG mapping`, then product can be taken from the user input. Probably doable when this script gets integrated to the srg_export.py which has the product as input already.